### PR TITLE
Use a workspace for all brb crates

### PR DESCRIPTION
**Pull request by dirvine**
To allow easier access to all brb algorithms.

TODO : 
[ ] move errors into separate files. 
[ ]Remove println! from code (use log) 
[ ] Fix tests 
[ ] Fix compilation problem 
[ ] CI/CD integration

### CODE REVIEW

File: brb_membership/LICENSE-BSD [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<<<<<<< HEAD
+Copyright 2019 MaidSafe.net limited.
+=======
+Copyright 2020 MaidSafe.net limited.
+>>>>>>> 1b80124... import dynamic membership code from brb crate (and before that bft-crdts)
```
Comment (davidrusu) [inline brb_membership/LICENSE-BSD:5]: Conflicted committed here

**Comment by dan-da** [comment]
It seems we've decided to go with individual brb_* crates for now due to CI/CD complexities that may arise when using a workspace.   Closing this.

**Comment by dan-da** [comment]
btw, for the record just in case we decide to second-guess ourselves about this, I have a more recent workspace branch that builds and runs tests cleanly and has some readme updates.

https://github.com/dan-da/brb/tree/brb_workspace_pr

File: brb_membership/LICENSE-BSD (hunk 1 above)
Comment (dan-da) [reply brb_membership/LICENSE-BSD:5]: thx, I've fixed this in a separate PR in the brb_membership crate.

**Pull request closed without merging**